content/zh-cn/docs/reference/networking/virtual-ips.md

```diff
@@ -0,0 +1,686 @@
+relevant Service.
+ -->
+
+你可以设置 `.spec.externalTrafficPolicy` 字段来控制从外部源路由的流量。
+有效值为 `Cluster` 和 `Local`。
+将字段设置为 `Cluster` 会将外部流量路由到所有准备就绪的端点，
+将字段设置为 `Local` 仅会将流量路由到本地节点上准备就绪的端点。
+如果流量策略为 `Local` 并且没有本地节点端点，
+那么 kube-proxy 不会转发与相关 Service 相关的任何流量。
+
+{{< note >}}
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+<!-- 
+If you enable the `ProxyTerminatingEndpoints`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+for the kube-proxy, the kube-proxy checks if the node
+has local endpoints and whether or not all the local endpoints are marked as terminating.
+If there are local endpoints and **all** of those are terminating, then the kube-proxy ignores
+any external traffic policy of `Local`. Instead, whilst the node-local endpoints remain as all
+terminating, the kube-proxy forwards traffic for that Service to healthy endpoints elsewhere,
+as if the external traffic policy were set to `Cluster`.
+ -->
+如果为 kube-proxy 启用了 `ProxyTerminatingEndpoints`
+[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)，
+kube-proxy 会检查节点是否具有本地端点以及是否所有本地端点都标记为终止。
+如果有本地端点并且 **所有** 本地端点都被标记为终止，则 kube-proxy 忽略 `Local` 的任何外部流量策略。
+相反，当余下的所有本地节点端点处于终止中时，
+kube-proxy 将该 Service 的流量转发到其他健康端点，
+就好像外部流量策略设置为 `Cluster` 一样。
+
+
+<!--
+This forwarding behavior for terminating endpoints exists to allow external load balancers to
+gracefully drain connections that are backed by `NodePort` Services, even when the health check
+node port starts to fail. Otherwise, traffic can be lost between the time a node is
+still in the node pool of a load balancer and traffic is being dropped during the
+termination period of a pod.
+ -->
+
+这种对处于终止中的端点的转发行为使得外部负载均衡器能优雅地排空由
+`NodePort` 服务支持的连接，即使在健康检查节点端口开始出现故障时也是如此。
+否则，在节点仍然在负载均衡器的节点池情况下，在 Pod 终止期间，流量可能会丢失。
+{{< /note >}}
+
+<!-- 
+### Internal traffic policy
+ -->
+
+### 内部流量策略 {#internal-traffic-policy}
+
+{{< feature-state for_k8s_version="v1.22" state="beta" >}}
+
+<!-- 
+You can set the `.spec.internalTrafficPolicy` field to control how traffic from
+internal sources is routed. Valid values are `Cluster` and `Local`. Set the field to
+`Cluster` to route internal traffic to all ready endpoints and `Local` to only route
+to ready node-local endpoints. If the traffic policy is `Local` and there are no
+node-local endpoints, traffic is dropped by kube-proxy.
+ -->
+你可以设置 `.spec.internalTrafficPolicy` 字段来控制从内部源路由的流量。
+有效值为 `Cluster` 和 `Local`。
+将字段设置为 `Cluster` 会将内部流量路由到所有准备就绪的端点，
+将字段设置为 `Local` 仅会将流量路由到本地节点准备就绪的端点。
+如果流量策略为 `Local` 但没有本地节点端点，那么 kube-proxy 会丢弃该流量。
+
+<!-- 
+## {{% heading "whatsnext" %}}
+ -->
+## {{% heading "whatsnext" %}}
+
+<!-- 
+To learn more about Services,
+read [Connecting Applications with Services](/docs/concepts/services-networking/connect-applications-service/).
+ -->
+要了解有关 Service 的更多信息，
+请阅读 [使用 Service 连接应用程序](/zh-cn/docs/concepts/services-networking/connect-applications-service/)。
```

Comment:
If you want, skip the redirect and link directly to `/zh-cn/docs/tutorials/services/connect-applications-service/`. This is OK even when the English version still relies on a redirect.

Reply:
I checked and found that  there was no Chinese version of  `/docs/concepts/services-networking/connect-applications-service/`.
Now it has been changed.